Installation.md

```diff
@@ -0,0 +1,267 @@
+## Setup ##
+
+### Notices before installation ###
+
+We assume you use [RVM](https://rvm.io/). If you don't, then replace `rvmsudo` with `sudo` during the installation process.
+
+:exclamation: RVM 1.17.x may have some [problems with rvmsudo](http://stackoverflow.com/questions/13765520/rvmsudo-command-not-working-properly).
+
+An X server is currently needed for browser tests to pass when using Selenium. `Xvfb` will do, but remember to set your `DISPLAY`.
+
+:exclamation: For testing environment PhantomJS (at least version 1.8.1) is required by Poltergeist gem to run headless browser tests. See [poltergeist](https://github.com/teampoltergeist/poltergeist) for documentation.
+
+Below is rough setup instructions, if you're running Ubuntu you can follow the [Ubuntu specific installation guide](#installation).
```

Comment:
Suggestion: `Below is a quick summary of the setup procedure. It is followed by detailed instructions for Ubuntu.`